### DISCUSSION TITLE
Placing the calculated unfolded state energy

### DISCUSSION THREAD

**Discussion by MuhammadAttaElhamouly** [RosettaCommons/rosetta#284] [General]
These documentations [using_ncaas_protein_peptide_interface_design](https://docs.rosettacommons.org/demos/latest/public/using_ncaas_protein_peptide_interface_design/README) and [unfolded_state_energy_calculator](https://docs.rosettacommons.org/demos/latest/public/unfolded_state_energy_calculator/README) stating the same directory `scoring/score_functions/unfolded/unfolded_state_residue_energies_mm_std `, to place the calculated unfolded state energy.

I have done that, but because I need to use the Ref2015 energy function, it seems that the Unfolded energy is not calculated, unfolded_state_residue_energies_mm_std  file isn't recalled to calculate. The Unfolded energy is one fixed value for all output.

I used the option `-unfolded_energies_file inputs/unfolded_mm_std`  to be able to use it. 

How to apply the unfolded energy for the NCAA only, not for the whole output, and how I can precisely use the Ref2015 energy function along with the unfolded state energy? Also, do I need to reweight the energy function due to this use, and if I need to do so, which weights I can use?

Note: Unfolded weight is different and ranging -0.9 to 1 according to different weights.

**Comment by garyd2002** [comment]
Don't add -mute all option in the command.

**For Example:**
UnfoldedStateEnergyCalculator.static.linuxgccrelease -ignore_unrecognized_res -extrachi_cutoff 0 -l pdb_list.lst -usec:residue_name XXX -no_optH true -detect_disulf false >&ufsec_log_xxx &

**Not** the following command recommended in the offical tutorial:
UnfoldedStateEnergyCalculator.static.linuxgccrelease -ignore_unrecognized_res -extrachi_cutoff 0 -l pdb_list_clean.lst -usec:residue_name XXX -mute all -unmute devel.UnfoldedStateEnergyCalculator -unmute protocols.jd2.PDBJobInputer -no_optH true -detect_disulf false >& ufsec_log_xxx &
